cmd/block-generator/generator/generate.go

```diff
@@ -607,7 +608,7 @@ func (g *generator) generateAssetTxnInternalHint(txType txID, sp sdk_types.Sugge
 
 	txn.Fee = 1000
 	if g.balances[senderIndex] < uint64(txn.Fee) {
-		fmt.Printf(fmt.Sprintf("\n\nthe sender account does not have enough algos for the transfer. idx %d, payment number %d\n\n", senderIndex, g.numPayments))
+		fmt.Printf(fmt.Sprintf("\n\nthe sender account does not have enough algos for the transfer. idx %d, asset type %v, num %d\n\n", senderIndex, actual, g.reportData[actual].GenerationCount))
```

Comment:
asset type -> transaction type?

Reply:
asset transaction type might be a little more accurate, but not transaction type. It's because asset transactions have multiple configurations to do different things (i.e. an asset xfer is used to optin/closeout/transfer).

Reply:
Fixed